package.yaml

```diff
@@ -0,0 +1,35 @@
+name: SecurityAdvisoriesBuilder-hs
+
+version: 0.1.0.0
+
+ghc-options: -Wall
+
+dependencies:
+  - base
+
+library:
+  source-dirs: src/
+  dependencies:
+    - bytestring >= 0.10.8.2
```

Comment:
Note: instead of putting the dependencies at root level, I specified them separately for `library` and `tests` - is that a good or a bad idea? Any pointers to how real world projects do this?